src/base/details/_base.scss

```diff
@@ -26,13 +26,23 @@ summary {
 }
 
 details {
-	padding-left: $details-identation;
-	padding-right: $details-identation;
+	padding-left: $details-indentation;
+	padding-right: $details-indentation;
 
 	> {
 		summary {
-			margin-left: -$details-identation;
-			margin-right: -$details-identation;
+			margin-left: -$details-indentation;
+			margin-right: -$details-indentation;
+
+			&.h1,
```

Comment:
We don't need to support .h1, let's just support our use cases (h2 to h6) knowing that .h1 shouldn't be used at all.